src/Congo.WebApi/Startup.cs

```diff
@@ -44,6 +44,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "Congo.WebApi v1"));
             }
 
+            app.UseCors(config => config.WithOrigins("*").AllowAnyMethod().AllowAnyHeader());
```

Comment:
This should be moved inside the `if (env.IsDevelopment())` block, so we only allow * in development.